src/cloud-api-adaptor/test/provisioner/libvirt/provision_common.go

```diff
@@ -337,6 +345,16 @@ func (lio *LibvirtInstallOverlay) Edit(ctx context.Context, cfg *envconf.Config,
 		}
 	}
 
+	if cred := os.Getenv("REGISTRY_CREDENTIAL_ENCODED"); cred != "" {
```

Comment:
Hi @mkulke , I'm about to send a PR with fixes for AWS and I was wondering how to create the auth.json. So likely I will reuse your work :)